external-api/api-tests/v1/email.api-test.ts

```diff
@@ -1,49 +1,95 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Please remove inline rules.

Reply:
I removed one rule by using Oscars solution, now our PRs will have conflict.

Other one is not new, it also present in other tests.